test/reproducibility_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+bazel clean
+bazel build test:play-routes
+routes_hash0=$(md5sum bazel-bin/test/play/routes/play_routes/router/Routes.scala | awk '{print $1}')
```

Comment:
Is there a way to test the reproducibility of the entire target rather than just these two files? If something nondeterministic gets introduced outside of these files, then we won't know.

Reply:
good point